src/configmanager.cpp

```diff
@@ -108,13 +108,15 @@ bool ConfigManager::load()
 		string[MYSQL_PASS] = getGlobalString(L, "mysqlPass", "");
 		string[MYSQL_DB] = getGlobalString(L, "mysqlDatabase", "forgottenserver");
 		string[MYSQL_SOCK] = getGlobalString(L, "mysqlSock", "");
+		string[HTTP_API_IP] = getGlobalString(L, "httpApiIp", "127.0.0.1");
```

Comment:
I guess @ranisalt  idea is to make it un-configurable. Here's it's just the default. I think the default should be fine, it should be possible to change the IP in case it's a Docker container and the proxy is in a different one ;)